pkg/storage/secret/keeper_model.go

```diff
@@ -0,0 +1,293 @@
+package secret
+
+import (
+	"encoding/json"
+	"fmt"
+	"time"
+
+	"github.com/google/uuid"
+	"github.com/grafana/grafana/pkg/apimachinery/errutil"
+	"github.com/grafana/grafana/pkg/apimachinery/utils"
+	secretv0alpha1 "github.com/grafana/grafana/pkg/apis/secret/v0alpha1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/types"
+)
+
+var (
+	ErrKeeperNotFound = errutil.NotFound("cloudmigrations.sessionNotFound").Errorf("Session not found")
```

Comment:
Nit: you can merge and then change this but should this naming be changed?

Reply:
oops yes 🙈